lib/vagrant-service-manager/service.rb

```diff
@@ -0,0 +1,35 @@
+# Loads all services
+Dir["#{File.dirname(__FILE__)}/services/*.rb"].each { |f| require_relative f }
+
+module Vagrant
+  module ServiceManager
+    SUPPORTED_BOXES = ['adb', 'cdk']
+
+    class Service
+      def initialize(app, env)
+        @app = app
+        @machine = env[:machine]
+        @ui = env[:ui]
+        @services = @machine.config.servicemanager.services.split(',').map(&:strip)
+        @docker_hook = Docker.new(@machine, @ui)
+        @openshift_hook = OpenShift.new(@machine, @ui)
+      end
+
+      def call(env)
+        @app.call(env)
+
+        if SUPPORTED_BOXES.include? @machine.guest.capability(:os_variant)
```

Comment:
@bexelbie : This line will only prevent other boxes not to trigger this plugin.
I would like to know your use case here please.